test/integration/scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
```

Comment:
These are almost identical, the only difference is the `"yarn test:integration $@"` instead of `"yarn test:rendering $@"`.
Feel free to suggest a less redundant solution.